doc/source/index.rst

```diff
@@ -30,3 +30,6 @@ bugs and fixing the code:
    developing
    changelog
    credits
+   
+print "hello world"
```

Comment:
This isn't valid python 3 syntax! It should be:

    print("hello world")

And probably "hello" should be capitalized.

@mauritsvanrees: make sure you never start working for @JoepGrispen!